docs/tutorial/notebook/train_ss_model_using_pytorch.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<blockquote> For this example, we will use the <code>RandLANet</code> model.</blockquote><p>This is already covered earlier. Avoid repetition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done.</p>